_topic_map.yml

```diff
@@ -2644,6 +2644,8 @@ Topics:
     File: ossm-support
   - Name: Service Mesh architecture
     File: ossm-architecture
+  - Name: Deployment models
```

Comment:
Not critical, but this nav name seems a bit different than the rest. Could it be "Service Mesh deployment models" to align more with the previous two items?